android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -206,6 +206,7 @@ fun ReadableMap.toAdItem(): AdItem? {
     return AdItem(
         sources = getArray("sources") ?.toMapList()?.mapNotNull { it?.toAdSource() }?.toTypedArray() ?: return null,
         position = getString("position") ?: "pre",
+        preloadOffset = getString("preloadOffset") ?: 0,
```

Comment:
Please read the value as a `Double` to match type. See related build error: https://github.com/bitmovin/bitmovin-player-react-native/actions/runs/10245493463/job/28340612293#step:4:274

```suggestion
        preloadOffset = getDoubleOrNull("preloadOffset") ?: 0,
```